packages/contracts/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Does this need to be in the repo? Could it be in your specific ignores?

Reply:
I googled that before commiting to confirm.
Following the advice in this answer: https://stackoverflow.com/a/57749909

Seems to make sense to me, since they're generic settings for any vs code user (nothing in this file is specific to myself), and doesn't hurt non-vscode users

Reply:
to be clear. the only purpose of this file is to make vscode run tests with `hardhat test` instead of `mocha`